pkg/cloud/scope/machine.go

```diff
@@ -180,29 +180,62 @@ func (m *MachineScope) SetAnnotation(key, value string) {
 	m.AWSMachine.Annotations[key] = value
 }
 
+// UserDataPrivacyRequested returns the computed value of whether or not
+// userdata should be stored using AWS Secrets Manager.
+func (m *MachineScope) UserDataPrivacyRequested() bool {
```

Comment:
```suggestion
func (m *MachineScope) ShouldEncryptUserData() bool {
```

Reply:
Renamed `UserdataPrivacyEnabled`

I've renamed `encryption` to `UserdataPrivacy` as that's more in line with what is actually happening.

Reply:
`ShouldSecureUserData`?